pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -394,26 +428,49 @@ message Capability {
     repeated string drop_capabilities = 2;
 }
 
+// SecurityContext holds security configuration that will be applied to a container.
+message SecurityContext {
+    // Capabilities to add or drop.
+    optional Capability capabilities = 1;
+    // If set, run container in privileged mode.
+    optional bool privileged = 2;
+    // The configurations for the container's namespaces.
+    // This will be used only if the container uses namespace for isolation.
+    optional NamespaceOption namespace_options = 3;
+    // Optional SELinux context to be applied.
+    optional SELinuxOption selinux_options = 4;
+    // The UID to run the the container process.
+    // Defaults to user specified in image metadata if unspecified.
+    optional int64 run_as_user = 5;
+    // Indicates that the container must run as a non-root user.
+    // If true, the runtime should validate the image at runtime to ensure that it
+    // does not run as UID 0 (root) and fail to start the container if it does.
+    // If unset or false, no such validation will be performed.
+    optional bool run_as_non_root = 6;
+    // If set, the root filesystem of the container is read-only.
+    optional bool readonly_rootfs = 7;
+    // A list of groups applied to the first process run in the container, in addition
+    // to the container's primary GID.  If unspecified, no groups will be added.
+    repeated int64 supplemental_groups = 8;
+    // A special supplemental group that applies to the container.
+    // Some volume types allow the Kubelet to change the ownership of that volume
+    // to be owned by the pod:
+    //
+    // 1. The owning GID will be the FSGroup
+    // 2. The setgid bit is set (new files created in the volume will be owned by FSGroup)
+    // 3. The permission bits are OR'd with rw-rw----
+    //
+    // If unset, the Kubelet will not modify the ownership and permissions of any volume.
+    optional int64 fs_group = 9;
```

Comment:
I don't think we need both `supplemental_groups` and `fs_groups` since they essentially mean the same thing for the runtime. Can we merge them? The same applies to the sandbox security context.